packages/atlas/src/components/_inputs/Slider/Slider.tsx

```diff
@@ -24,7 +24,13 @@ export const Slider: React.FC<SliderProps> = ({
   return (
     <StyledRcSlider
       value={value}
-      onChange={onChange}
+      onChange={(value) => {
```

Comment:
What happened here? Why we didn't need it before?

Reply:
Yeah, previously we received a single value, now it's an array
![Screenshot 2022-06-14 at 10 22 46](https://user-images.githubusercontent.com/12646744/173531283-62df905f-d910-4b14-abab-d31bc61c95f7.jpg)
![Screenshot 2022-06-14 at 10 28 22](https://user-images.githubusercontent.com/12646744/173531293-6376b599-cf61-4891-95c2-eba037653e8b.jpg)